packages/leancode_lint/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+# 1.0.2
```

Comment:
Should be a `1.0.1+1` imo since it adds nothing to the package itself

Reply:
That's a good point, I haven't thought of it.